src/TouristInfoCards.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import HotelImage from "./images-folder/hotel.jpeg";
+
+function TouristInfoCards(props) {
```

Comment:
I love that we have broken this down into a small component! So easy to see what is going on 😄 
But we have called it Cards over Card would be my only issuee, there is only one card in here. Best to make it as clear as possible what a component is actually doing!